module-system/sov-modules-macros/tests/dispatch/derive_event.rs

```diff
@@ -12,12 +12,35 @@ struct Runtime<C: Context>
 {
     pub first: first_test_module::FirstTestStruct<C>,
     pub second: second_test_module::SecondTestStruct<C>,
+    pub third: third_test_module::ThirdTestStruct<C, u32>,
 }
 
 fn main() {
     // Check to see if the runtime events are getting initialized correctly
-    let _event = RuntimeEvent::<DefaultContext>::first(first_test_module::Event::FirstModuleEnum1(10));
-    let _event = RuntimeEvent::<DefaultContext>::first(first_test_module::Event::FirstModuleEnum2);
-    let _event = RuntimeEvent::<DefaultContext>::first(first_test_module::Event::FirstModuleEnum3(vec![1;3]));
-    let _event = RuntimeEvent::<DefaultContext>::second(second_test_module::Event::SecondModuleEnum);
+    {
+        let event = RuntimeEvent::<DefaultContext>::first(first_test_module::Event::FirstModuleEnum1(10));
+        assert_eq!(event.get_key_string(), "first-FirstModuleEnum1");
```

Comment:
Will this generate an expected string even if  `Event` is `struct/tuple/array` instead of an enum type?